Framework/PythonInterface/plugins/algorithms/IndirectTransmission.py

```diff
@@ -173,7 +173,7 @@ def _get_efixed(self, workspace):
         except ValueError:
             # If that fails then get it by taking from group of all detectors
             wsHandle = mtd[workspace]
-            spectra_list = list(range(0, wsHandle.getNumberHistograms()))
+            spectra_list = list(range(1, wsHandle.getNumberHistograms() + 1))
```

Comment:
Ah yes I see. I think in this case it would then be better to just use SumSpectra without any arguments to sum everything. @louisemccann Do you think this makes sense?